src/SIL.XForge.Scripture/appsettings.Development.json

```diff
@@ -7,12 +7,9 @@
       "Hangfire": "Information"
     }
   },
-  "Site": {
-    "Domain": "beta.scriptureforge.local"
-  },
   "Site": {
     "Name": "Scripture Forge (beta)",
-    "Domain": "beta.scriptureforge.org",
+    "Domain": "beta.scriptureforge.local",
     "SmtpServer": "mail.ecgroup.in",
```

Comment:
We probably want to find a better default value for this. This will allow actual e-mail sending from inside ecgroup, but isn't helpful elsewhere.